src/EnergyPlus/AirLoopHVACDOAS.hh

```diff
@@ -174,7 +174,7 @@ namespace AirLoopHVACDOAS {
         // default constructor
         AirLoopDOAS() // constructor
             : SumMassFlowRate(0.0), PreheatTemp(-999.0), PrecoolTemp(-999.0), PreheatHumRat(-999.0), PrecoolHumRat(-999.0), SizingMassFlow(0.0),
-              SizingCoolOATemp(-999.0), SizingCoolOAHumRat(-999.0), HeatOutTemp(0.0), HeatOutHumRat(0.0), m_AirLoopDOASNum(0), m_OASystemNum(0),
+              SizingCoolOATemp(-999.0), SizingCoolOAHumRat(-999.0), HeatOutTemp(999.0), HeatOutHumRat(999.0), m_AirLoopDOASNum(0), m_OASystemNum(0),
```

Comment:
If per chance there are no design days, or no summer or no winter days, is there a check to quit or do something reasonable if one of these is still -999 or 999?